src/it/resources/com/google/checkstyle/test/chapter4formatting/rule4822declaredwhenneeded/InputFormattedDeclaredWhenNeeded.java

```diff
@@ -0,0 +1,752 @@
+package com.google.checkstyle.test.chapter4formatting.rule4822declaredwhenneeded;
+
+import java.util.Calendar;
+import java.util.Locale;
+import java.util.TimeZone;
+
+/** Some javadoc. */
+public class InputFormattedDeclaredWhenNeeded {
+
+  private static int test1 = 0;
+
+  static {
+    int b = 0;
+    int d = 0;
+    { // violation ''block lcurly' has incorrect indentation level 4, expected level should be 6.'
+      d = ++b;
+    } // violation ''block rcurly' has incorrect indentation level 4, expected level should be 6.'
+  }
+
+  static {
+    int c = 0;
+    int a = 3;
+    int b = 2;
+    { // violation ''block lcurly' has incorrect indentation level 4, expected level should be 6.'
+      a = a + b;
+      c = b;
+    } // violation ''block rcurly' has incorrect indentation level 4, expected level should be 6.'
+    { // violation ''block lcurly' has incorrect indentation level 4, expected level should be 6.'
+      c--;
+    } // violation ''block rcurly' has incorrect indentation level 4, expected level should be 6.'
```

Comment:
these are false positives from our side and it is a known issue https://github.com/checkstyle/checkstyle/issues/14294